acra/configuring-maintaining/optimisations/acrastructs_vs_acrablocks.md

```diff
@@ -0,0 +1,44 @@
+---
+title: AcraStructs vs AcraBlocks
+weight: 3
+---
+
+# AcraStructs vs AcraBlocks
+
+There are two different "crypto containers" AcraServer can use to store encrypted value in.
+Then, these containers are stored in database and decrypted back by AcraServer when needed.
+
+`AcraStruct` is the main crypto container for the client-side encryption.
```

Comment:
I'd leave `` `backticks` `` flair for code examples and other verbatim uses. I think using PascalCase is enough to make AcraStruct to stand out as a term.

On a side note, while looking for a way to escape those, here's what I found out about GitHub parser:

### <code><code><code><code><code><code><code><code><code><code><code><code><code><code><code><code>code</code></code></code></code></code></code></code></code></code></code></code></code></code></code></code></code>

By the way, the string above is

    `` `backticks` ``